benefits/core/admin.py

```diff
@@ -17,21 +17,69 @@
 GOOGLE_USER_INFO_URL = "https://www.googleapis.com/oauth2/v3/userinfo"
 STAFF_GROUP_NAME = "Cal-ITP"
 
+logger.debug("Register models with admin site")
+admin.site.register(models.PemData)
```

Comment:
The spreadsheet lists the `text_secret_name` field in `PemData` as readable only by superusers.

Reply:
Oh, somehow I had the impression non-superusers wouldn't see `PemData`s at all, but you're totally right - the spreadsheet definitely says they can read everything but `text_secret_name`.

Fixed this in https://github.com/cal-itp/benefits/pull/2224/commits/3f3b37d7bf93ada92c9fc1bc628627e12bbf273b

Reply:
Oh, I see -- you had it so they couldn't see the model at all. I think that's better :sweat_smile: sorry! It doesn't make a lot of sense for non-superusers to see that model.

Reply:
No worries! I think it might've just been the way I filtered the spreadsheet that made me think that, but I also like hiding the `PemData`s altogether better. I'll drop the commit and update the spreadsheet

Reply:
Done and done